apps/fz_http/test/fz_http_web/acceptance/admin_test.exs

```diff
@@ -647,11 +648,13 @@ defmodule FzHttpWeb.Acceptance.AdminTest do
       session =
         session
         |> click(Query.css("button[aria-label=\"close\"]"))
+        |> assert_path(~p"/settings/account")
```

Comment:
Assert path is not a wait-wrapped assertion which can fail on CI or other places with limited resources. We should move them below 'assert_el' to make it less flaky